src/coreclr/debug/di/process.cpp

```diff
@@ -11152,7 +11155,41 @@ void CordbProcess::FilterClrNotification(
     }
 }
 
+#if defined(TARGET_WINDOWS) && defined(TARGET_AMD64)
```

Comment:
> I'd recommend using an ifdef named with the feature rather than the OS or architecture. Otherwise when we need to port features in the future it is harder to determine which code matters. You would need to add the new define if one doesn't already exist.

Thanks for the suggestion.  @hoyosjs suggested the same.  It looks like CET is built into the product now and does not have a define.  I believe it is beyond the scope of this PR to add a new CET ifdef.  I will create an issue to track this request.

Reply:
I don't think there is much ceremony here, you could add the define with less text than it takes to file the tracking issue : )

In src\coreclr\clrdefinitions.cmake add:
```
if (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_TARGET_ARCH_AMD64)
    add_definitions(-DFEATURE_CET)
endif (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_TARGET_ARCH_AMD64)
```

Reply:
Yes I had already experimented with adding a feature flag for out-of-process setthreadcontext - https://github.com/tommcdon/runtime/commit/b8ad42d4182d3e7a8212bcd6a6d28180b4b082e1.  I had originally felt the new define wasn't necessary because 1) we need architecture specific ifdef's under the feature ifdef's even though the feature flag is only enabled for one platform/one architecture, 2) the runtime's CET implementation doesn't have a feature flag so seems confusing to a add a feature flag that only works on part of the runtime's codebase, 3) the feature is a runtime check (not compile time) for x64 Windows.  With that said I don't feel strongly enough to push back, so I'll move forward with adding that change :).

Reply:
The suggestion is largely based on the assumption that even if CET is windows x64 for now, its probably only a matter of time before we need to support it on another OS or another architecture. When that time comes if devs can search for FEATURE_CET in the source or enable the ifdef and see what breaks that is much easier than having to do lots of code review figuring out which #ifdef WINDOWS X64 code is CET and which is not : )